keywords/couchbaseserver.py

```diff
@@ -342,9 +345,9 @@ def create_buckets(self, bucket_names, ipv6=False):
         per_bucket_ram_mb = self.get_ram_per_bucket(len(bucket_names))
 
         for bucket_name in bucket_names:
-            self.create_bucket(bucket_name, per_bucket_ram_mb, ipv6)
+            self.create_bucket(cluster_config, bucket_name, per_bucket_ram_mb, ipv6)
 
-    def create_bucket(self, name, ram_quota_mb=1024, ipv6=False):
+    def create_bucket(self, cluster_config, name, ram_quota_mb=1024, ipv6=False):
```

Comment:
same as above, it is used in install_sync_gateway.py, cluster.py and test_multiple_accels.py  which gets effected due to this change. YOu can pass default value as None to avoid those files failing

Reply:
same as above.